components/admin/admin-notifications.tsx

```
@@ -200,57 +284,99 @@
                         key={notification.id}
-                        className={`p-4 hover:bg-muted/50 cursor-pointer transition-colors ${
-                          !notification.isRead ? "bg-blue-50 dark:bg-blue-950/20" : ""
+                        className={`relative p-4 hover:bg-muted/30 cursor-pointer transition-all duration-200 border-l-4 ${
+                          getNotificationPriorityColor(notification.type)
+                        } ${
+                          !notification.isRead 
+                            ? "bg-primary/5 hover:bg-primary/10" 
+                            : "hover:bg-muted/50"
                         }`}
                         onClick={() => handleNotificationClick(notification)}
                       >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)